src/MAUI/Maui.Samples/Samples/Data/ManageFeatures/ManageFeatures.xaml.cs

```diff
@@ -0,0 +1,439 @@
+        {
+            // This code needs to work with the UI, so it needs to run on the UI thread.
+            Microsoft.Maui.ApplicationModel.MainThread.BeginInvokeOnMainThread(() =>
+            {
+                // Get the relevant field from the table.
+                ServiceFeatureTable table = (ServiceFeatureTable)sender;
+                Field typeDamageField = table.Fields.First(field => field.Name == AttributeFieldName);
+
+                // Get the domain for the field.
+                CodedValueDomain attributeDomain = (CodedValueDomain)typeDamageField.Domain;
+
+                // Update the ComboBox with the attribute values.
+                DamageTypePicker.ItemsSource = attributeDomain.CodedValues.Select(codedValue => codedValue.Name).ToList();
+            });
+        }
+
+        private async void OnComboBoxSelectionChanged(object sender, EventArgs e)
+        {
+            // Clear all currently hooked GeoViewTapped events.
+            MyMapView.GeoViewTapped -= MapView_Tapped_CreateFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_DeleteFeature;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateAttribute;
+            MyMapView.GeoViewTapped -= MapView_Tapped_UpdateGeometry;
+
+            // Reset UI elements.
+            _damageLayer.ClearSelection();
+            _selectedFeature = null;
+            DeleteButton.IsVisible = false;
+            DamageTypePicker.IsVisible = false;
+
+            // Hold a variable for the Picker's selected value.
+            Picker picker = (Picker)sender;
+
+            try
+            {
+                if (picker.SelectedIndex == 0)
+                {
+                    // Update the label.
+                    InstructionLabel.Text = "Tap on the map to create a new feature.";
+
+                    // Update the currently hooked GeoViewTapped event.
+                    MyMapView.GeoViewTapped += MapView_Tapped_CreateFeature;
+
+                }
+                else if (picker.SelectedIndex == 1)
+                {
+                    // Update the label.
+                    InstructionLabel.Text = "Tap an existing feature to delete it.";
+
+                    // Update the currently hooked GeoViewTapped event.
+                    MyMapView.GeoViewTapped += MapView_Tapped_DeleteFeature;
+                }
+                else if (picker.SelectedIndex == 2)
+                {
+                    // Update the label.
+                    InstructionLabel.Text = "Tap an existing feature to edit its attribute.";
+
+                    // Update the currently hooked GeoViewTapped event.
+                    MyMapView.GeoViewTapped += MapView_Tapped_UpdateAttribute;
+                }
+                else if (picker.SelectedIndex == 3)
+                {
+                    // Update the label.
+                    InstructionLabel.Text = "Tap an existing feature to select it, tap the map to move it to a new position.";
```

Comment:
Maybe something like "Tap an existing feature to select it and again to move it." to make it less wordy?